runners/spark/src/main/java/org/apache/beam/runners/spark/translation/TransformTranslator.java

```diff
@@ -776,7 +777,7 @@ public Boolean call(Tuple2<TupleTag<V>, WindowedValue<?>> input) {
     EVALUATORS.put(HadoopIO.Write.Bound.class, writeHadoop());
     EVALUATORS.put(ParDo.Bound.class, parDo());
     EVALUATORS.put(ParDo.BoundMulti.class, multiDo());
-    EVALUATORS.put(GroupByKey.GroupByKeyOnly.class, gbk());
+    EVALUATORS.put(GroupByKeyOnly.class, gbk());
```

Comment:
Good point. I didn't catch that this runner does use the expansion. I will move some bits around (more things have to be public). I will still use `util/` as a temporary holding pen for the pieces for now, I think.